storage/sqlite/sqlite.go

```diff
@@ -0,0 +1,270 @@
+// Если база данных существует, то подключается к существующей базе денных.
+// Если база данных не существует, то создает новую БД.
+// New The function returns a reference to the database. It is necessary to pass the path to the database through the DBFile variable
+// If the database exists, it connects to the existing database.
+// If the database does not exist, it creates a new database.
+func New(DBFile string) (*Storage, error) {
+
+	_, err := os.Stat(DBFile)
+	var install bool
+	if err != nil {
+		install = true
+	}
+
+	if install {
+		_, err = os.Create(DBFile)
+		if err != nil {
+			log.Fatal(err)
+		}
+	}
+
+	//db, err := sqlx.Connect("sqlite3", DBFile) -- для работы необходим  TDM-GCC
+	db, err := sqlx.Connect("sqlite3", DBFile)
+
+	if err != nil {
+		fmt.Println(err)
+		return nil, err
+	}
+	if install {
+		_, err = db.Exec("CREATE TABLE scheduler (id  INTEGER PRIMARY KEY AUTOINCREMENT, date VARCHAR, title VARCHAR(128) NOT NULL, comment VARCHAR, repeat VARCHAR(128) )")
+		if err != nil {
+			log.Panic(err)
+			return nil, err
+		}
+		_, err = db.Exec("CREATE INDEX ID_Date ON scheduler (date)")
+		if err != nil {
+			log.Panic(err)
+			return nil, err
+		}
+
+	}
+
+	return &Storage{db: db}, nil
+}
+
+// Close method to close the database
+// Close метод для закрытия базы данных
+func (s *Storage) Close() {
+	s.db.Close()
+}
+
+// AddTask method adds tasks to the database. The method interacts with http-servet.handlers.PostTask .
+// Takes data as input and returns id in string format or error
+// AddTask метод добавляет задачи в базу данных. Метод взаимодействует с ручкой http-servet.handlers.PostTask .
+// Принимает на вход данные и возвращает id в формате string или ошибку
+func (s *Storage) AddTask(date string, title string, comment string, repeat string) (string, error) {
+	stmt, err := s.db.Prepare("INSERT INTO scheduler(date, title,comment,repeat) VALUES(?, ?,?,?)")
+	if err != nil {
+		return "", fmt.Errorf("failed to create a request for database update", err)
+	}
+
+	res, err := stmt.Exec(date, title, comment, repeat)
+	if err != nil {
+		return "", fmt.Errorf("failed to INSERT a request for database update", err)
+	}
+
+	id, err := res.LastInsertId()
+	if err != nil {
+		return "", fmt.Errorf("Failed to get last insert id:", err)
+	}
+	idString := strconv.Itoa(int(id))
+	return idString, nil
+
+}
+
+// GetTasks method returns the specified in NumberOfOutTasks from the database The method interacts with the handle http-servet.handlers.GetTasksHundler
+// Returns a response in the form of a slice of handlers.Task structures or an error
+// GetTasks метод возвращает из базы данных указанное в NumberOfOutTasks Метод взаимодействует с ручкой http-servet.handlers.GetTasksHundler
+// Возвращает ответ в форме слайса структур handlers.Task или ошибку
+func (s Storage) GetTasks(NumberOfOutTasks int, tasks []handlers.Task) ([]handlers.Task, error) {
```

Comment:
Не очень понятно зачем массив тасок на входе
```suggestion
func (s Storage) GetTasks(NumberOfOutTasks int) ([]handlers.Task, error) {
```